index.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+    
+<link>
+
+<!--@import-->
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,500;0,700;1,700&display=swap" rel="stylesheet">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+
+    <h1>Hello world</h1>
+    <h2>We are software developers</h2>
+    <script  src="js/index.js"></script>
+    <div class="container">
+      <h1>freeCodeCamp Survey Form</h1>
+      <p>
+        Thank you for taking the time to help us improve the platform
+      </p>
+    </div>
+
+    <form class="form">
+      <div class="form-input">
+        <label>Name</label>
+        <input  class="inputs" type="text" placeholder="Enter your Name"/>
+      </div>
+      <div class="form-input">
+        <label>Email</label>
+        <input  class="inputs" type="text" placeholder="Enter your Email"/>
+      </div>
+      <div class="form-input">
+        <label>Age (optional)</label>
+        <input  class="inputs" type="text" placeholder="Enter your Age"/>
+      </div>
+      <div class="form-input">
+        <label>Which Option best describes your current role</label>
```

Comment:
Add for in your labels. Labels should usually have for property ie. `<label for='username' >` for easy accesablity.